xchange-binance/src/main/java/org/knowm/xchange/binance/service/BinanceAccountService.java

```diff
@@ -201,8 +195,16 @@ public TradeHistoryParams createFundingHistoryParams() {
           (Long)
               exchange.getExchangeSpecification().getExchangeSpecificParametersItem("recvWindow");
 
+      Integer limit = null;
+
+      if (params instanceof TradeHistoryParamLimit) {
+        TradeHistoryParamLimit hpl = (TradeHistoryParamLimit) params;
+        limit = hpl.getLimit();
+      }
+
       boolean withdrawals = true;
       boolean deposits = true;
+      boolean airdrops = false;
```

Comment:
as binances documentations states those "dividends" wont always be airdrops, in fact they will be mostly stake rewards:

https://www.financemagnates.com/cryptocurrency/news/binance-launches-staking-service-for-coins-held-in-its-wallets/

until we implement better parsing of the `enInfo` field I'd suggest to add a new value to:

```
public class FoundingRecord {

  /** Enum representing funding transaction type */
  public enum Type {
    WITHDRAWAL(false),
    DEPOSIT(true),
    AIRDROP(true),
    /** Used for inflows that are not a regular users deposit and are either different the the inflows defined above or their nature could not have been deduced from the exchanges response */
    OTHER_INFLOW(true);
    /** Used for outflows that are not a regular users withdrawal and are either different the the outflows defined above or their nature could not have been deduced from the exchanges response */
    OTHER_INFLOW(false);
``` 

and name the boolean here differenty

```
boolean otherInflow = true;
```